src/Model/Photo.php

```diff
@@ -460,6 +460,29 @@ public static function importProfilePhoto($image_url, $uid, $cid, $quit_on_error
 		if ($Image->isValid()) {
 			$Image->scaleToSquare(300);
 
+			$filesize = strlen($Image->asString());
+			$maximagesize = DI::config()->get('system', 'maximagesize');
+			if (!empty($maximagesize) && ($filesize > $maximagesize)) {
+				Logger::info('Avatar exceeds image limit', ['uid' => $uid, 'cid' => $cid, 'maximagesize' => $maximagesize, 'size' => $filesize, 'type' => $Image->getType()]);
+				if (($Image->getType() == 'image/gif')) {
```

Comment:
Extraneous parentheses.
```suggestion
				if ($Image->getType() == 'image/gif') {
```

Reply:
Oh yeah. This had been some leftover from my various tests.